Lesson01-Favorite-Toys/T01.02-Exercise-DisplayToyList/app/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion '25.0.2'
+    compileSdkVersion 27
+    buildToolsVersion '27.0.0'
```

Comment:
As I know Android Studio by default suggests to use 26.0.2, it stable version. API 27 is Android Oreo 8.1, it's not released yet. Can we use stable version?

Reply:
Ok.